egs/wham/DPRNN/README.md

```diff
@@ -3,6 +3,8 @@
 |      |   task    |kernel size|chunk size|batch size|SI-SNRi(dB) | SDRi(dB)|
 |:----:|:---------:|:---------:|:--------:|:--------:|:----------:|:-------:|
 | Paper| sep_clean |    16     |     100  |     -    |    15.9    |   16.1  |
-| Here | sep_clean |    16     |     100  |     8    |    17.7    |  17.9   |
+| Here | sep_clean |    16     |     100  |     8    |    17.7    |  18.0   |
```

Comment:
'sar': 18.779807098538395,                                                                                                   
 'sar_imp': -130.39664151902417,                                                                                             
 'sdr': 18.26455730164243,                                                                                                
 'sdr_imp': 18.113515087336932,                                                                                               
 'si_sdr': 17.866566573185224,                                                                                               
 'si_sdr_imp': 17.867717301372167,

Reply:
this is what I obtain with ksz=16 , I ve checked and is same as your modifications also relu mask